web/Dockerfile

```diff
@@ -1,6 +1,12 @@
 # base image
 FROM node:20.11.0-alpine AS base
 
+RUN apk add --no-cache tzdata
+
+# set timezone
+ENV TZ UTC
```

Comment:
I think it should be included in the production stage.

Reply:
Resolved.